fondant/cli.py

```diff
@@ -28,91 +27,77 @@
     run_explorer_app,
 )
 from fondant.pipeline import Pipeline
+from fondant.runner import DockerRunner
 
-logging.basicConfig(level=logging.INFO)
-cli = argparse.ArgumentParser(description="Fondant CLI")
-subparsers = cli.add_subparsers()
+logger = logging.getLogger(__name__)
 
 
 def entrypoint():
+    """Entrypoint for the fondant CLI."""
+    parser = argparse.ArgumentParser(description="Fondant CLI")
+    subparsers = parser.add_subparsers()
+    register_explore(subparsers)
+    register_compile(subparsers)
+    register_run(subparsers)
+
     sys.path.append(".")
-    args = cli.parse_args()
+
+    args = parser.parse_args(sys.argv[1:])
```

Comment:
```suggestion
    args = parser.parse_args()
```
I don't think explicitly providing the arguments is necessary.